src/cfnlint/rules/mappings/Used.py

```diff
@@ -0,0 +1,63 @@
+"""
+  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this
+  software and associated documentation files (the "Software"), to deal in the Software
+  without restriction, including without limitation the rights to use, copy, modify,
+  merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+  permit persons to whom the Software is furnished to do so.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+  INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+  PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+  HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+from cfnlint import CloudFormationLintRule
+from cfnlint import RuleMatch
+
+
+class Used(CloudFormationLintRule):
+    """Check if Mappings are used anywhere in the template"""
+    id = 'W7001'
+    shortdesc = 'Check if Mappings are Used'
+    description = 'Making sure the mappings defined are used'
+    tags = ['base', 'conditions']
+
+    def match(self, cfn):
+        """Check CloudFormation Mappings"""
+
+        matches = list()
+        ref_mappings = list()
+
+        mappings = cfn.template.get('Mappings', {})
+
+        if mappings:
+
+            # Get all "FindInMaps" that reference a Mapping
+            maptrees = cfn.search_deep_keys('Fn::FindInMap')
+
+            for maptree in maptrees:
+
+                if isinstance(maptree[-1], list):
+                    ref_mappings.append(maptree[-1][0])
+                else:
+                    ref_mappings.append(maptree[-1])
+
+#            # Get resource's Mappings (I don't think this applies?)
+#            for _, resource_values in cfn.get_resources().items():
```

Comment:
This can be removed indeed. The only thing that can reference a Maping is a FindInMap function

Reply:
Cool! I'll pull this code out.  I thought I had :)